src/backend/optimizer/plan/aqumv.c

```diff
@@ -458,11 +458,11 @@ answer_query_using_materialized_views(PlannerInfo *root,
 			/*
 			 * Update pathkeys which may be changed by qp_callback.
 			 * Set belows if corresponding feature is supported.
-			 * sort_pathkeys
 			 * distinct_pathkey
 			 * window_pathkeys
 			 */
 			root->group_pathkeys = subroot->group_pathkeys;
+			root->sort_pathkeys = subroot->sort_pathkeys;
```

Comment:
what's diffrence with `query_pathkeys` ?

Reply:
> what's diffrence with `query_pathkeys` ?

`query_pathkeys` is the desired pathkeys for query_planner(),  see details in 
https://github.com/postgres/postgres/blob/REL_14_STABLE/src/backend/optimizer/plan/planner.c#L3057